app/javascript/components/subject/CaseStatus.js

```diff
@@ -70,9 +99,12 @@ class CaseStatus extends React.Component {
     let current = this.state.showCaseStatusModal;
     this.setState({
       showCaseStatusModal: !current,
+      confirmedOrProbable: this.props.patient.case_status === 'Confirmed' || this.props.patient.case_status === 'Probable',
       apply_to_group: false,
```

Comment:
GitHub is frustrating and doesn't allow me to directly comment on a block of unchanged code so I'm commenting here. I think this state variable should be change from `apply_to_group` to `applyToHousehold` for clarity.

Reply:
apply_to_group is the variable name that is used in the db and what we send off in the submit (see below).  this is used anywhere the HoH can apply to the whole household, so if we want to change it I think we should change it everywhere - frontend and backend.

![Screen Shot 2020-10-05 at 4 11 31 PM](https://user-images.githubusercontent.com/35042815/95127120-7bf88100-0725-11eb-91b5-16b2f4ccb6e2.png)

Reply:
Yeah I'm aware, this is honestly part of a larger issue I already have been tracking to have consistent variable naming conventions since right now it's all over the place on the frontend, so out of scope of this PR.